CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ different versioning scheme, following the Haskell community's
 * C# NuGet version: TBD (minor bump needed)
 * C# Comm NuGet version: TBD (minor bump needed)
 
+### C++ & `gbc` ###
+
+* Remove FieldTemplate details from field reflection type.
+
```

Comment:
Minor: there's a `gbc` section right below where this would make more sense. Consider also duplicating the entry in the C++ section as well.